src/app/plugins/addclasschans.plugin.ts

```diff
@@ -12,7 +12,7 @@ export class AddClassChannelsPlugin extends Plugin {
   public name: string = 'Add many classes';
   public description: string = 'creates a bunch of class channels';
   public usage: string = 'addclasschans';
-  public pluginAlias = [];
+  public override pluginAlias = [];
   public permission: ChannelType = ChannelType.Admin;
```

Comment:
Why is only the alias an override for all plugins?

Reply:
Because a lot of contributors have been overriding the default value given in the base class. The other fields aren't marked with override because they're required.